CMakeLists.txt

```diff
@@ -134,6 +134,7 @@ check_type_size("long" SIZEOF_LONG)
 # Require C++14
 set(CMAKE_CXX_STANDARD 14)
 set(CMAKE_CXX_STANDARD_REQUIRED TRUE)
+set(CMAKE_CXX_EXTENSIONS FALSE)
```

Comment:
I don't think I like this. It inhibits optional but beneficial features that could be useful for the generated code.

Reply:
agree with you, removed